src/languages/lsl.js

```diff
@@ -0,0 +1,133 @@
+/*
+    Language: Linden Scripting Language
+    Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+    Author: Builder's Brewery <buildersbrewery@gmail.com>
+    Category: common, scripting
+*/
+
+function(hljs) {
+    return {
+        case_insensitive: false,
+        aliases: ['lsl'],
+        illegal: ':',
+        contains: [
+            {
+                className: 'string',
+                begin: '"',
+                end: '"'
+            },
+            {
+                className: 'comment',
+                begin: '//',
+                end: '$'
+            },
+            {
+                className: 'comment',
+                begin: '/\\*',
+                end: '\\*/'
+            },
```

Comment:
Please do use the helper to define comments: http://highlightjs.readthedocs.io/en/latest/language-guide.html#comments